src/javascript/_common/base/currency_base.js

```diff
@@ -79,11 +79,20 @@ const CryptoConfig = (() => {
 
 const getMinWithdrawal = currency => (isCryptocurrency(currency) ? (getPropertyValue(CryptoConfig.get(), [currency, 'min_withdrawal']) || 0.002) : 1);
 
-// returns in a string format, e.g. '0.00000001'
-const getMinTransfer = currency => {
-    const min_transfer = getPropertyValue(currencies_config, [currency, 'transfer_between_accounts', 'limits', 'min']) || getMinWithdrawal(currency);
-    const decimals     = getDecimalPlaces(currency);
-    return min_transfer.toFixed(decimals); // we need toFixed() so that it doesn't display in scientific notation, e.g. 1e-8 for currencies with 8 decimal places
+/**
+ * Returns the transfer limits for the account.
+ * @param currency
+ * @param which
+ * @returns numeric
```

Comment:
seems like sometimes object is returned (e.g. when which is missing) and sometimes undefined is returned. can we really say it returns numeric then?

Reply:
Good point. I'll add a check to see if no `which` is passed and `transfer_limits.min` is also undefined,  throw error.